scripts/github-actions/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG BASE_IMAGE=quay.io/pypa/manylinux2010_x86_64
```

Comment:
May be this should live under `docker/github-actions/`?

Reply:
We tried moving `scripts/github-actions` to `.github/scripts`. Let's hope it works! :crossed_fingers: